package-build.json

```diff
@@ -27,6 +27,6 @@
     },
     "peerDependencies": {
       "vue": "^3.0.0",
-      "primeicons": "^5.0.0"
+      "primeicons": "6.0.1"
```

Comment:
This should probably be `"^6.0.0"` or `"^5.0.0 || ^6.0.0"` (or `5 - 6`)? Else it will require this very specific version.